bitcoin/src/pow.rs

```diff
@@ -1077,21 +1077,36 @@ impl kani::Arbitrary for U256 {
     }
 }
 
+/// In test code, U256s are a pain to work with, so we just convert Rust primitives in many places
 #[cfg(test)]
-mod tests {
-    use super::*;
+pub mod test_utils {
+    use crate::pow::{Target, Work, U256};
+
+    #[allow(missing_docs)]
+    pub fn convert_u64_to_work(u: u64) -> Work {
```

Comment:
Perhaps call the module `convert` and the the functions `u64_to_work`, then at the call sites use `convert::u64_to_work`, and import with `use crate::pow::convert`.

Just an idea.

Reply:
That would look nicer. I do think that having something about testing in the module name would make it immediately clear that these are meant for testing only though, instead of having to look at the declaration and seeing the test flag. Any reason we shouldn't limit it to just testing-related code?

Reply:
Definitely only test code, and fair point about having 'test' in the name. Lets stick with your original module name (even though we try not to use 'util'). Perhaps drop the `convert_`.